frontend/src/lib/api.ts

```diff
@@ -2742,7 +2742,7 @@ async function handleFetch(url: string, method: string, fetcher: () => Promise<R
         error = e
     }
 
-    apiStatusLogic.findMounted()?.actions.onApiResponse(response, error)
+    apiStatusLogic.findMounted()?.actions.onApiResponse(response?.clone(), error)
```

Comment:
This is needed because it calls `resonse.json()` so downstream functions can't pull out the body response.